src/index.html

```diff
@@ -5,8 +5,410 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = ""
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a class="logoMoyo" href="#home">
+        <img src="images/logo.png" alt="moyo-logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#APPLE" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#SAMSUNG" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphone" class="nav__link">смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#notebook"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadjet" class="nav__link">гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablet" class="nav__link">планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog body__catalog">
+
+      <div class="catalog__container">
+        <div class="card catalog__card" data-qa="card">
```

Comment:
It's not necessary to make `main` parent element to `container` and `card`, and our task requires
`main class="main"` you can add you own classes but `main` must be.